homeassistant/components/sensor/__init__.py

```diff
@@ -90,6 +91,7 @@
     DEVICE_CLASS_POWER,  # power (W/kW)
     DEVICE_CLASS_POWER_FACTOR,  # power factor (%)
     DEVICE_CLASS_VOLTAGE,  # voltage (V)
+    DEVICE_CLASS_FREQUENCY,  # frequency (Hz)
```

Comment:
Adding new device classes will need a corresponding PR in developer docs:
https://github.com/home-assistant/developers.home-assistant/blob/master/docs/core/entity/sensor.md

Reply:
This PR is changing our entity model. Before we can review this PR there needs to be approval in a discussion in our architecture repository.

https://github.com/home-assistant/architecture/discussions

https://developers.home-assistant.io/docs/core/entity#changing-the-entity-model